stock_picking_product_generate_sequence/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+##############################################################################
+# Copyright (c) 2022 brain-tec AG (https://braintec.com)
+# All Right Reserved
+#
+# See LICENSE file for full licensing details.
+##############################################################################
+{
+    'name': 'Stock Picking Product Generate sequence',
+    'version': '15.0.1.0.0',
+    'summary': 'Add an automated sequence in product of stock picking',
+    'category': 'Inventory',
+    'author': 'brain-tec AG',
```

Comment:
[Major] If it will be contributed to the OCA it needs them as author as well

Reply:
I added